.github/workflows/deploy-published-releases.yaml

```
@@ -35,8 +35,9 @@ jobs:
         run: |-
           cp package.json LICENSE README.md build/
           cd build
+          jq package.json '.devDependencies = {}' > ./package.json
```

This will result in an empty file. The `jq` pipe should be used _instead_ of the copy:
```suggestion
          cp LICENSE README.md build/
          jq package.json '.devDependencies = {}' > ./build/package.json
          cd build
```